conf/brain.config

```diff
@@ -0,0 +1,72 @@
+// ─────────────────────────────────────────────────────────────────────────────
+// Global Parameters
+// ─────────────────────────────────────────────────────────────────────────────
+params {
+    config_profile_name        = 'Brain Image Library Analysis Ecosystem'
+    config_profile_description = 'A public resource enabling researchers to deposit, analyze, mine, share, and interact with large brain image datasets.'
+    config_profile_url         = 'https://www.brainimagelibrary.org'
+    config_profile_contact     = 'Ivan Cao-Berg (@icaoberg)'
+
+    // General resource boundaries (used in profile as well)
+    max_memory = 2.9.Tb
+    max_cpus   = 80
+    max_time   = 48.h
+
+    // Ignore iGenomes when not needed
+    igenomes_ignore = true
+    igenomesIgnore  = true
+}
+
+// ─────────────────────────────────────────────────────────────────────────────
+// Singularity Configuration
+// ─────────────────────────────────────────────────────────────────────────────
+singularity {
+    enabled     = true
+    autoMounts  = true
+    runOptions  = '''
+        --home $HOME:/bil/users/$USER \
+        --bind /scratch \
+        --bind /local
+    '''
+}
+
+// ─────────────────────────────────────────────────────────────────────────────
+// Executor Configuration
+// ─────────────────────────────────────────────────────────────────────────────
+executor {
+    queueSize       = 10
+    submitRateLimit = '5 sec'
+}
+
+// ─────────────────────────────────────────────────────────────────────────────
+// Profiles
+// ─────────────────────────────────────────────────────────────────────────────
+profiles {
+    compute {
```

Comment:
I this a reason why this is a separate profile - given you have a single entry?

Reply:
we have multiple partitions but only adding the public at the moment.

Reply:
I see - however you don't need a specific profile for a specific partition - you can tell Nextflow to dynamically pick the queue based on whatever variables you want to use rather than having to explicitly request it.

So I would move everything in this `profile` into the base config, and when you need to add additional partitions we can update the profile to be dynamic.